lib/minio/client.rb

```diff
@@ -147,8 +147,14 @@ def list_objects(bucket_name, folder_path, max_keys: 1000)
     objects
   end
 
-  def send_request(method, uri, body = nil)
-    headers = @signer.build_headers(method, uri, body, @creds, REGION)
+  def set_lifecycle_policy(bucket_name, policy_id, expiration_days)
+    policy = "<LifecycleConfiguration><Rule><ID>#{policy_id}</ID><Status>Enabled</Status><Filter></Filter><Expiration><Days>#{expiration_days}</Days></Expiration></Rule></LifecycleConfiguration>"
```

Comment:
Thinking about it, this is one wiggle away from some kind of injection attack. I do appreciate not bringing in a whole XML serializer though. Can we have a runtime regexp check immediately before to ensure that policy_id and expiration_days aren't "weird"? Being alphanumeric (and no angle brackets) or something simple like that suffices. Does `/\A[\w\s]+\z/` suffice?

```ruby
[2] clover-development(main)> /\A[\w\s]+\z/.match("hello world")
=> #<MatchData "hello world">
[3] clover-development(main)> /\A[\w\s]+\z/.match("<hello world")
=> nil
```